src/queryResult/queryResultWebViewController.ts

```diff
@@ -305,6 +305,10 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
         state: qr.QueryResultWebviewState,
         xmlPlans: string[],
     ) {
+        sendActionEvent(
```

Comment:
`sendActionEvent`) (and `sendErrorEvent()`) both have optional props that can be set: `additionalMeasurements` (most numeric metrics) and `additionalProperties` (everything else).

Capturing something like the number of plans as a metric here would be good so we can see if, say, people are frequently viewing plans for 25 queries at once, or if we always get errors when it hits some threshold.